JPS_Ontology/ontology/ontosealevel/OntoSeaLevel.csv

```diff
@@ -28,7 +26,7 @@ Monument,Class,https://raw.githubusercontent.com/beAWARE-project/ontology/master
 Scenario,Class,,,,,,Scenario that can act as input to a forecasting level for forecasting sea level.,https://www.theworldavatar.com/kg/ontosealevel/,Scenario
 SharedSocioEconomicPathway,Class,Scenario,IS-A,,,,A climate change scenario of projected socioeconomic changes.,https://www.theworldavatar.com/kg/ontosealevel/,Shared Socioeconomic Pathway
 measuredAt,Object Property,,,LocalMeanSeaLevel,http://www.opengis.net/ont/geosparql#Feature,exactly 1,A relationship between the local mean sea level and the reference point for the local mean sea level measurement.,https://www.theworldavatar.com/kg/ontosealevel/,measured at
-measuredIn,Data Property,,,SeaLevelMeasurement,http://www.w3.org/2001/XMLSchema#gYear UNION http://www.w3.org/2001/XMLSchema#date,,A relationship between a sea level measurement and the time of measurement.,https://www.theworldavatar.com/kg/ontosealevel/,measured in
+measuredIn,Data Property,,,SeaLevelMeasurement,http://www.w3.org/2001/XMLSchema#gYear UNION http://www.w3.org/2001/XMLSchema#date UNION http://www.w3.org/2001/XMLSchema#dateTime UNION http://www.w3.org/2006/time#Instant,,A relationship between a sea level measurement and the time of measurement.,https://www.theworldavatar.com/kg/ontosealevel/,measured in
```

Comment:
Would you be able to explain in a readme file the design/intended use of this relationship, given that a sea level measurement is a time series (and hence the apparent redundancy)?

Reply:
My initial thinking was that it could be the case that we only have one datapoint for a forecast or actual measurement. And that having this relationship, it would be easier to query for cases with one datapoint.

I've given more thoughts to it. Yeah, this is redundant, as a single data point can be instantiated as a time series with 1 entry.